docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -105,6 +105,80 @@ Complete the following steps to install Grafana using DEB or the standalone bina
 1. Copy and paste the code from the [download page](/grafana/download) into your command line and run.
 
+## Install Grafana as a standalone binary
+
+Complete the following steps to install Grafana using the standalone binaries:
+
+1. Navigate to the [Grafana download page](/grafana/download).
+1. Select the Grafana version you want to install.
+   - The most recent Grafana version is selected by default.
+   - The **Version** field displays only tagged releases. If you want to install a nightly build, click **Nightly Builds** and then select a version.
+1. Select an **Edition**.
+   - **Enterprise:** This is the recommended version. It is functionally identical to the open-source version but includes features you can unlock with a license if you so choose.
+   - **Open Source:** This version is functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
+1. Depending on which system you are running, click the **Linux** or **ARM** tab on the [download page](/grafana/download).
+1. Copy and paste the code from the [download page](/grafana/download) page into your command line and run.
+1. Create a user account for Grafana on your system:
+
+   ```shell
+   sudo useradd -r -s /bin/false grafana
+   ```
+
+1. Move the unpacked binary to `/usr/local/grafana`:
+
+   ```shell
+   sudo mv <your_grafana_download> /usr/local/grafana
+   ```
+
+1. Change the owner of `/usr/local/grafana` to Grafana users:
+
+   ```shell
+   sudo chown -R grafana:users /usr/local/grafana
+   ```
+
+1. Create a Grafana server `systemd` unit file:
+
+   ```shell
+   sudo touch /etc/systemd/system/grafana-server.service
+   ```
+
+1. Add the following to the unit file in a text editor of your choice:
+
+   ```ini
+   [Unit]
+   Description=Grafana Server
+   After=network.target
+
+   [Service]
+   Type=simple
+   User=grafana
+   Group=users
+   ExecStart=/usr/local/grafana/bin/grafana server --config=/usr/local/grafana/conf/grafana.ini --homepath=/usr/local/grafana
+   Restart=on-failure
+
+   [Install]
+   WantedBy=multi-user.target
+   ```
+
+1. Use the binary to manually start the Grafana server:
+
+   ```shell
+   /usr/local/grafana/bin/grafana-server --homepath /usr/local/grafana
+   ```
+
+   {{% admonition type="note" %}}
+   Manually invoking the binary in this step automatically creates the `/usr/local/grafana/data` directory, which needs to be created and configured before the installation can be considered complete.
+   {{% /admonition %}}
+
+1. Press `CTRL+C` to stop the Grafana server.
+1. Change the owner of `/usr/local/grafana` to Grafana users again to apply the ownership to the newly created `/usr/local/grafana/data` directory:
+
+   ```shell
+   sudo chown -R grafana:users /usr/local/grafana
+   ```
+
+1. [Configure the Grafana server to start at boot time using systemd]({{< relref "../../start-restart-grafana#configure-the-grafana-server-to-start-at-boot-using-systemd" >}}).
```

Comment:
We should avoid relrefs: https://grafana.com/docs/writers-toolkit/write/shortcodes/#relref